gradebook.py

```diff
@@ -65,3 +65,29 @@ def view_grades(self, instructor, course_id):
             print_warning("No grades have been entered for this course yet.")
         
         input("Press enter to continue.")
+
+    def sort_courses(self, arrangement_type):
+        # Checks if dictionary is empty
+        if not self.grades:
+            print("Grades are empty. Please add a grade")
+            return
+
+        if arrangement_type is not 'a' or arrangement_type is not 'd':
```

Comment:
Do you mean: `arrangement_type != 'a' or arrangement_type != 'd'` ?

> is checks for identity - if the two variables point to the exact same object.
> == checks for equality - if the two variables point at values are equal.  

-- Check this article for details: https://adamj.eu/tech/2020/01/21/why-does-python-3-8-syntaxwarning-for-is-literal/